src/Containers.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import cockpit from 'cockpit';
+import Listing from '../lib/cockpit-components-listing.jsx';
+import ContainerDetails from './ContainerDetails.js';
+import Dropdown from './DropDown.js';
+
+const _ = cockpit.gettext;
+
+const navigateToContainer = (container) => {
+	cockpit.location.go([container.ID]);
+}
+
+//TODO
+const deleteContainer = (event, container) => {
+	return undefined;
+}
+
+//TODO
+const stopContainer = (container) => {
+	return undefined;
+}
+
+//TODO
+const startContainer = (container) => {
+	return undefined;
+}
+
+//TODO
+const restartContainer = (container) => {
+	return undefined;
+}
+
+
+const renderRow = (containersStats, container) => {
+	const isRunning = !!container.State.Running;
+	//TODO: check container.Image == container.ImageID
+	const image = container.ImageName;
+	const state = container.State.Status;
+
+	let columns = [
+		{ name: container.Name, header: true },
+		image,
+		container.Config.Entrypoint === "" ? container.Config.Cmd.join(" ") : container.Config.Cmd.join(""),
+		// TODO: CpuUsage
+		"",
+		// TODO: MemoryUsage, MemoryLimit
+		"",
+		state,
```

Comment:
This should still be i18n'ed, but I'm happy for this to be a `// TODO: i18n` comment for now.